content/reference/compose-file/services.md

```diff
@@ -1222,6 +1222,7 @@ There is a performance penalty for applications that swap memory to disk often.
 - `none`: Turns off all container networking.
 - `host`: Gives the container raw access to the host's network interface.
 - `service:{name}`: Gives the containers access to the specified service only. For more information, see [Container networks](/manuals/engine/network/_index.md#container-networks).
+- `container:{name}`: Gives the service container access to the specified container.
```

Comment:
I guess that `service:name` and `container:name` both do the same thing, right? Just that one of them references the service name and the other the container name? Could this be clarified?